mon-pix/app/components/tutorial-panel.js

```diff
@@ -1,36 +1,28 @@
-import { classNames } from '@ember-decorators/component';
-import { computed } from '@ember/object';
-import Component from '@ember/component';
-import classic from 'ember-classic-decorator';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 
-@classic
-@classNames('tutorial-panel')
 export default class TutorialPanel extends Component {
-  hint = null;
-  tutorials = null;
+  @tracked hint = null;
+  @tracked tutorials = null;
```

Comment:
ces deux lignes sont inutiles non? Puisque que:
- `hint` et `tutorials` sont donnés en arguments depuis `comparison-window.hbs`: `<TutorialPanel @hint={{answer.correction.hint}} @tutorials={{answer.correction.tutorials}} />`
- je pense que en déclarant `@tracked`, on accèderait à `hint` et `tutorials` via `this` et on pas `this.args`